internal/livelint/livelint.go

```diff
@@ -1,23 +1,27 @@
 package livelint
 
 import (
+	"net/http"
+
 	tea "github.com/charmbracelet/bubbletea"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 )
 
 type Livelint struct {
-	k8s    kubernetes.Interface
-	config *rest.Config
-	ui     *tea.Program
+	k8s        kubernetes.Interface
+	config     *rest.Config
+	ui         *tea.Program
+	httpClient *http.Client
```

Comment:
Should we just call this `http`? Then using it would feel more like the standard library's `http`.

Reply:
Agreed. Done!